examples/preprocessing/plot_count_featurizer.py

```diff
@@ -0,0 +1,138 @@
+"""
+=========================================================
+Using CountFeaturizer to featurize frequencies
+=========================================================
+
+Shows how to use CountFeaturizer to transform some categorical variables
+into a frequency feature. CountFeaturizer can often be used to reduce
+training time, classification time, and classification error.
```

Comment:
I would replace the last sentence with:

 CountFeaturizer can be used as an alternative to one-hot encoding for non-linear estimators that do not work efficiently on high cardinality categorical variables such as ensemble of trees (random forests and gradient boosted trees).